app/src/view/MainView.java

```diff
@@ -0,0 +1,91 @@
+package view;
+
+import java.awt.BorderLayout;
+import java.awt.CardLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.awt.event.ActionListener;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+import javax.swing.border.EmptyBorder;
+
+public class MainView extends JPanel
+{
+	
+	private CardLayout cardLayout = new CardLayout();
+	
+	private HRManagementView hrManagementView = new HRManagementView();
+	private FinancialView financialView = new FinancialView();
+	private MedicalView medicalView = new MedicalView();
+	
+	private JPanel menuContainer = new JPanel(new GridLayout(1, 3));
+	private JPanel pageContainer = new JPanel(cardLayout);
+	
+	private JButton module1Button = new JButton("Gestiune Resurse Umane");
+	private JButton module2Button = new JButton("Financiar Contabil");
+	private JButton module3Button = new JButton("Medical");
+	
+	public final static String HR_TAB = "HR";
+	public final static String FINANCIAL_TAB = "FINANCIAL";
+	public final static String MEDICAL_TAB = "MEDICAL";
+	
+	public MainView()
+	{
+		initComponent();
+	}
+
+	public void initComponent() {
+		setLayout(new BorderLayout());
+		
+		menuContainer.setPreferredSize(new Dimension(getWidth(), 40));
+		
+		menuContainer.add(module1Button);
+		menuContainer.add(module2Button);
+		menuContainer.add(module3Button);
+		
+		pageContainer.add(HR_TAB, hrManagementView);
+		pageContainer.add(FINANCIAL_TAB, financialView);
+		pageContainer.add(MEDICAL_TAB, medicalView);
+		
+		add(menuContainer, BorderLayout.NORTH);
+		add(pageContainer, BorderLayout.CENTER);
+	}
+	
+	public void switchTab(String tabName)
+	{
+		cardLayout.show(pageContainer, tabName);
```

Comment:
Could check for whether the tab you want to switch to exists.

Reply:
This is unnecessary, as discussed.